cmd/s3-benchmark/main.go

```diff
@@ -270,6 +315,39 @@ func Benchmark(client Client, bucket string, size memory.Size, count int, durati
 	return measurement, nil
 }
 
+// ListBenchmark runs list buckets, folders and files benchmarks on bucket
+func ListBenchmark(client Client, bucket string, listsize int, count int, duration time.Duration) (Measurement, error) {
+	log.Print("Benchmarking list")
+ 	defer fmt.Println()
+ 	measurement := Measurement{}
+	//measurement.Size = listsize
+ 	{ // list folders
```

Comment:
Do repeated measurements here like above, and check for test duration.

```
	start := time.Now()
	for k := 0; k < count; k++ {
		if time.Since(start) > duration {
			break
		}
		fmt.Print(".")
```

That way the P values mean at least something.